x-pack/solutions/observability/plugins/metrics_data_access/server/lib/create_search_client.ts

```diff
@@ -6,12 +6,35 @@
  */
```

Comment:
There are duplicates of `createSearchClient` and the entire `kibana_framework_adapter`. I don't know why, this needs to be addessed separately